app/src/main/res/values/strings.xml

```diff
@@ -559,10 +559,10 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="quest_religion_for_wayside_shrine_title">What religion is represented at this shrine?</string>
     <string name="quest_parking_fee_title">Do you have to pay to park here?</string>
     <string name="quest_surface_value_unhewn_cobblestone">Unhewn cobblestone</string>
-    <string name="quest_parking_access_title">Is it restricted who may park here?</string>
-    <string name="quest_parking_access_private">It is private</string>
-    <string name="quest_parking_access_yes">It is open to the public (with or without fee)</string>
-    <string name="quest_parking_access_customers">It is only for customers</string>
+    <string name="quest_parking_access_title2">Who may park here? Parking may be free or paid.</string>
```

Comment:
I noticed the [wiki `key:access` page](https://wiki.openstreetmap.org/wiki/Key:access) specifies,

> Access values describe **legal** permissions/restrictions […] It does not describe common or typical use, even if signage is generally ignored.

I think this phrasing conveys that more clearly:

```suggestion
    <string name="quest_parking_access_title2">Who is allowed to park here? Parking may be free or paid.</string>
```